### PR TITLE
Added External Candidate pages

### PR DESCRIPTION
**Pull request by trudge-64**

### CODE REVIEW

**Review by ccg-oharan — APPROVED** [review]

**Review by ccg-oharan — APPROVED** [review]

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 1]
```diff
@@ -127,52 +127,39 @@
...
         <div class="row">
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/SO.jpg" alt="Photo of a ships' officer" class="img-responsive thumbnail mrgn-bttm-sm" />
+                    <img src="~/img/SO.jpg" alt="Photo of a ships' officer" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["ShipOfficerSO"]</h6></figcaption>
                 </figure>
             </div>
         </div>
-        <div class="row">
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:163]: No l.onger needed

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 2]
```diff
@@ -10,57 +10,57 @@
         <div class="row">
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <a href="Compare_Position"><img src="~/Theresa/img/AS.jpg" alt="Photo of engineering room" class="img-responsive thumbnail mrgn-bttm-sm" /></a>
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:13]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 3]
```diff
@@ -10,57 +10,57 @@
...
                 <figure>
-                    <a href="Compare_Position"><img src="~/Theresa/img/AS.jpg" alt="Photo of engineering room" class="img-responsive thumbnail mrgn-bttm-sm" /></a>
+                    <a href="Compare_Position"><img src="~/img/AS.jpg" alt="Photo of engineering room" class="img-responsive thumbnail mrgn-bttm-sm" /></a>
                     <figcaption class="text-center"><h6>@Localizer["AdminAS"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/ENG.jpg" alt="Photo of engineering room" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:20]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 4]
```diff
@@ -10,57 +10,57 @@
...
+                    <img src="~/img/ENG.jpg" alt="Photo of engineering room" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["EngineeringENG"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of supplies on a port" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:27]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 5]
```diff
@@ -10,57 +10,57 @@
...
+                    <img src="~/img/CS-clerical.jpg" alt="Photo of three CCG Members having a discussion" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["ClericalCR"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/mcts.jpg" alt="Photo of computer systems" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:34]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 6]
```diff
@@ -10,57 +10,57 @@
...
+                    <img src="~/img/mcts.jpg" alt="Photo of computer systems" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["ComputerCS"]</h6></figcaption>
                 </figure>
             </div>
         </div>
         <div class="row">
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:42]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 7]
```diff
@@ -10,57 +10,57 @@
...
+                    <img src="~/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["EconomicsEC"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:49]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 8]
```diff
@@ -10,57 +10,57 @@
...
+                    <img src="~/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["EducationED"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:56]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 9]
```diff
@@ -10,57 +10,57 @@
...
+                    <img src="~/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["EduSupportEU"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:63]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 10]
```diff
@@ -69,25 +69,25 @@
         <div class="row">
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:72]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 11]
```diff
@@ -69,25 +69,25 @@
...
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of" class="img-responsive thumbnail mrgn-bttm-sm" />
+                    <img src="~/img/EX-executive.jpg" alt="Photo of Executive members of the CCG" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["ExecutiveEX"]</h6></figcaption>
                 </figure>
             </div>
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/GL.jpg" alt="Photo of a welder" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:78]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 12]
```diff
@@ -69,25 +69,25 @@
...
+                    <img src="~/img/GL.jpg" alt="Photo of a welder" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["GLabourGL"]</h6></figcaption>
                 </figure>
             </div>
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/GS.jpg" alt="Photo of table setup with condiments" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:84]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 13]
```diff
@@ -69,25 +69,25 @@
...
+                    <img src="~/img/GS.jpg" alt="Photo of table setup with condiments" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["GServicesGS"]</h6></figcaption>
                 </figure>
             </div>
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/EG.jpg" alt="Photo of table setup with condiments" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:90]: Removed Theresa/ no longer needed for images

Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:90]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 14]
```diff
@@ -97,28 +97,28 @@
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:100]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 15]
```diff
@@ -97,28 +97,28 @@
...
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
+                    <img src="~/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["LibraryLS"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/LI.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:107]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 16]
```diff
@@ -97,28 +97,28 @@
...
+                    <img src="~/img/LI.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["LightkeepersLI"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:114]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 17]
```diff
@@ -97,28 +97,28 @@
...
+                    <img src="~/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["NursingNU"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:121]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 18]
```diff
@@ -127,52 +127,39 @@
         <div class="row">
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:130]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 19]
```diff
@@ -127,52 +127,39 @@
...
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
+                    <img src="~/img/blank.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["ProgramAdminPM"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/PG.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:137]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 20]
```diff
@@ -127,52 +127,39 @@
...
+                    <img src="~/img/PG.jpg" alt="Photo of landmass and anchor" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["PurchasingPG"]</h6></figcaption>
                 </figure>
             </div>
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/blank.jpg" alt="Photo of landmass offshore" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:143]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 21]
```diff
@@ -127,52 +127,39 @@
...
+                    <img src="~/img/RO-radio.jpg" alt="Photo of MCTS Officer Robert Kennedy working his standard shift at Victoria MCTS" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["RadioRO"]</h6></figcaption>
                 </figure>
             </div>
 
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/SC.jpg" alt="Photo of crewmembers" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:150]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/All_Positions.cshtml [hunk 22]
```diff
@@ -127,52 +127,39 @@
...
+                    <img src="~/img/SC.jpg" alt="Photo of crewmembers" class="img-responsive thumbnail mrgn-bttm-sm" />
                     <figcaption class="text-center"><h6>@Localizer["ShipCrewSC"]</h6></figcaption>
                 </figure>
             </div>
         </div>
         <div class="row">
             <div class="col-sm-3 mrgn-bttm-md">
                 <figure>
-                    <img src="~/Theresa/img/SO.jpg" alt="Photo of a ships' officer" class="img-responsive thumbnail mrgn-bttm-sm" />
```
Comment (trudge-64) [inline Web/Pages/Mockups/All_Positions.cshtml:158]: Removed Theresa/ no longer needed for images

File: Web/Pages/Mockups/Compare_Position.cshtml [hunk 23]
```diff
@@ -9,7 +9,7 @@
 <section role="contentinfo">
 
     <h2>Select a level and position</h2>
-    <figure><img src="~/Theresa/img/Positions-full.jpg" alt="Photo of " class="img-responsive  mrgn-bttm-md" /></figure>
```
Comment (trudge-64) [inline Web/Pages/Mockups/Compare_Position.cshtml:12]: Removed Theresa/ no longer needed for images

**Comment by trudge-64** [comment]
Going to update Mockup pages again

**Pull request closed without merging**